.github/workflows/develop_freeze_comment.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will comment on Pull Requests that are submitted while develop
+# is frozen during the week of release. Skips any Pull Requests that have the
+# label 'ignore-freeze'.
+# This workflow should be enabled only while develop is frozen.
+
+name: Develop Freeze PR Comment
+
+on:
+  # Trigger the workflow on pull request on develop branch
+  pull_request:
+    types:
+      - opened
+      - reopened
+    branches:
+      - develop
+
+jobs:
+  freeze-comment:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'ignore-freeze') }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: PR Comment
+        uses: github-actions-up-and-running/pr-comment@v1.0.1
```

Comment:
Per go/github-actions, we need to pin to a specific commit for third-party actions.

Reply:
Done.